superset/overlays/superset-init-job/db-init-job.yaml

```
@@ -32,11 +32,11 @@ spec:
         - name: superset-data
           persistentVolumeClaim:
             claimName: superset-data
-        - configMap:
+        - Secret:
```

```suggestion
        - secret:
```

I think capital S is not a valid Kubernetes spec here